src/extension.ts

```diff
@@ -40,6 +42,68 @@ export async function activate(context: ExtensionContext) {
 
 }
 
+function setupDecorators(context: ExtensionContext) {
+	let timeout: NodeJS.Timer | undefined = undefined;
+	const compiletimeDecorator = vscode.window.createTextEditorDecorationType({
+		borderWidth: '1px',
+		borderStyle: 'solid',
+		overviewRulerColor: 'blue',
+		overviewRulerLane: vscode.OverviewRulerLane.Right,
+		light: {
+			// this color will be used in light color themes
+			borderColor: 'darkblue'
+		},
+		dark: {
+			// this color will be used in dark color themes
+			borderColor: 'lightblue'
+		}
```

Comment:
Not a big fan of borders, but since there are not so many compiletime functions it is maybe ok.

Alternatives would be:

- Make `@compiletime` bold
- Use Gutter icons
- Use code lenses